flocker/docs/prompt_patched.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+# coding: utf-8
+
+
+from docutils import nodes
+from docutils.parsers import rst
+
+from pygments import highlight
```

Comment:
I think that if we import something directly it should have a version-pinned dependency in setup.py